packages/dx-react-demos/src/bootstrap3/basic/table-cell-template.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Grid,
+  TableView,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-bootstrap3';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../demo-data/generator';
+
+const HighlightedTableCell = ({ value, style, colSpan }) => (
+  <td
+    style={{
+      backgroundColor: 'red',
+      ...style,
+    }}
+    colSpan={colSpan}
+  >
+    <span style={{ color: 'white' }}>{value}</span>
+  </td>
+);
+
+HighlightedTableCell.propTypes = {
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]).isRequired,
+  style: PropTypes.object,
+  colSpan: PropTypes.number,
+};
+
+HighlightedTableCell.defaultProps = {
+  style: {},
+  colSpan: 1,
+};
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { name: 'region', title: 'Region' },
+        { name: 'sector', title: 'Sector' },
+        { name: 'channel', title: 'Channel' },
+        { name: 'customer', title: 'Customer' },
+        { name: 'product', title: 'Product' },
+        { name: 'amount', title: 'Sale Amount' },
+      ],
+      rows: generateRows({ columnValues: globalSalesValues, length: 14 }),
+    };
+
+    this.tableCellTemplate = ({ column, value, style, colSpan }) => {
```

Comment:
Why do we put it into this? It's pure and can be declared outside of the Demo component.